Makefile

```diff
@@ -89,6 +89,7 @@ appstore:
 	--exclude=mkdocs.yml \
 	--exclude=Makefile \
 	--exclude=node_modules \
+	--exclude=openapi.json \
```

Comment:
Revert as this file should be packaged as it is used by https://apps.nextcloud.com/apps/ocs_api_viewer